bpfctl/src/main.rs

```diff
@@ -173,13 +174,18 @@ async fn main() -> anyhow::Result<()> {
     // URI is ignored on UDS, so any parsable string works.
     let address = String::from("http://localhost");
     let unix = endpoint.unix.clone();
-    let channel = Endpoint::try_from(address)?
+    match Endpoint::try_from(address)?
         .connect_with_connector(service_fn(move |_: Uri| UnixStream::connect(unix.clone())))
-        .await?;
-
-    match execute_request(&cli.command, channel).await {
-        Ok(_) => return Ok(()),
-        Err(e) => eprintln!("Error = {e:?}"),
+        .await
+    {
+        Ok(channel) => {
+            info!("Using UNIX socket as transport");
+            match execute_request(&cli.command, channel).await {
+                Ok(_) => return Ok(()),
+                Err(e) => eprintln!("Error = {e:?}"),
```

Comment:
If we are using `log`, maybe we want to change all instance of `eprintln!` to either `warn!` or `error!`, seems a little confusing when one would be used over the other to me, WDYT?